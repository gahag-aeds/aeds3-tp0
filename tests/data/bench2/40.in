40 20
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 
Add 3 15
Sum 27 39
Add 2 14
Sum 6 18
Sub 17 29
Max 11 23
Sub 25 37
Min 25 37
Add 9 21
Max 8 20
Sub 2 14
Min 1 13
Sub 13 25
Max 23 35
Add 8 20
Max 23 35
Sub 15 27
Sum 18 30
Add 13 25
Max 5 17
