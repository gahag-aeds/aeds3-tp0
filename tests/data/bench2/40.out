677
608
66
5
74
17
92
92
684
89
